packages/core/src/conversation/ConversationService/ConversationService.ts

```diff
@@ -430,21 +437,43 @@ export class ConversationService {
     const mlsConversations = foundConversations.filter(isMLSConversation);
 
     //check all the established conversations' epoch with the core-crypto epoch
-    for (const {qualified_id: qualifiedId, group_id: groupId, epoch} of mlsConversations) {
-      try {
-        //if conversation is not established or epoch does not match -> try to rejoin
-        if (!(await this.isMLSConversationEstablished(groupId)) || !(await this.matchesEpoch(groupId, epoch))) {
-          this.logger.log(
-            `Conversation (id ${qualifiedId.id}) was not established or it's epoch number was out of date, joining via external commit`,
-          );
-          await this.joinByExternalCommit(qualifiedId);
-        }
-      } catch (error) {
-        this.logger.error(
-          `There was an error while handling epoch mismatch in MLS conversation (id: ${qualifiedId.id}):`,
-          error,
+    await Promise.allSettled(
```

Comment:
Is there a particular reason why we use `allSettled`. Do we expect some to fail?

Reply:
I didn't want one failure stop handling other conversations, but actually all logic inside `handleConversationEpochMismatch` is wrapped with try/catch, we can simply use `Promise.all` ;) will update it.